Packs/ctf01/Integrations/CortexXDRIRCTF/CortexXDRIRCTF.py

```diff
@@ -0,0 +1,5981 @@
+                    "referenced_resource": None,
+                    "region": None,
+                    "regionid": None,
+                    "registryhive": None,
+                    "registrykey": None,
+                    "registryvalue": None,
+                    "registryvaluetype": None,
+                    "relatedcampaign": None,
+                    "relatedendpoints": None,
+                    "relatedreport": None,
+                    "reminder": "0001-01-01T00:00:00Z",
+                    "remote_cid": None,
+                    "renderedhtml": None,
+                    "reportedemailcc": None,
+                    "reportedemailfrom": None,
+                    "reportedemailmessageid": None,
+                    "reportedemailorigin": None,
+                    "reportedemailsubject": None,
+                    "reportedemailto": None,
+                    "reporteremailaddress": None,
+                    "residentnotificationoption": None,
+                    "residentsemailaddress": None,
+                    "resolution_comment": "",
+                    "resolution_status": "STATUS_020_UNDER_INVESTIGATION",
+                    "resolution_status_modified_ts": 1674646611059,
+                    "resource_sub_type": None,
+                    "resource_type": None,
+                    "resourcename": None,
+                    "resourcetype": None,
+                    "riskrating": None,
+                    "riskscore": None,
+                    "rulename": None,
+                    "runStatus": "pending",
+                    "samaccountname": None,
+                    "scenario": None,
+                    "secretarynotification": None,
+                    "sectorofaffectedparty": None,
+                    "selectedindicators": None,
+                    "sensorip": None,
+                    "severity": "SEV_030_MEDIUM",
+                    "sha1": None,
+                    "sha512": None,
+                    "signature": None,
+                    "similarincidents": None,
+                    "similarincidentsdbot": None,
+                    "sizenumberofemployees": None,
+                    "sizeturnover": None,
+                    "skuname": None,
+                    "skutier": None,
+                    "sla": 0,
+                    "source_insert_ts": 1674646578590,
+                    "sourcecategory": None,
+                    "sourcecreatedby": None,
+                    "sourcecreatetime": None,
+                    "sourceexternalips": None,
+                    "sourcegeolocation": None,
+                    "sourceid": None,
+                    "sourceinstance": None,
+                    "sourcenetworks": None,
+                    "sourcepriority": None,
+                    "sourcestatus": None,
+                    "sourceupdatedby": None,
+                    "sourceurgency": None,
+                    "srcos": None,
+                    "ssdeep": None,
+                
```

Comment:
Sasha this raw alert is crazy. Lets remove some of the keys (a lot of them).

Reply:
We are using this raw list to fill out the alerts associated with RDP incidents based on the username - indeed we can remove some of the fields, but truly not sure if that can impact the code. Would you like to cover that with me ?